function-service-impl/src/test/java/com/future/function/service/impl/feature/scoring/ScoringMediatorServiceImplTest.java

```diff
@@ -98,6 +104,12 @@
   @Mock
   private RoomService roomService;
 
+  @Mock
+  private UserService userService;
+
+  @Mock
+  private ExecutorService executorService;
```

Comment:
I assume this mock is never verified due to the use of `CompletableFuture.runAsync`?